Update/hima_001.txt

```diff
@@ -1805,7 +1805,7 @@ void main()
 //　今から推定４８時間前、建設大臣の孫である、建設省幹部の子息が誘拐されたらしい＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　今から推定４８時間前、建設大臣の孫である、建設省幹部の子息が誘拐されたらしい。",
-		   NULL, "It seemed that approximately 48 hours earlier, the Minister of Construction's grandson and those of his senior staff members were kidnapped.", GetGlobalFlag(GLinemodeSp));
+		   NULL, "It seemed that approximately 48 hours earlier, the Minister of Construction's grandson, who was also the son of one of his senior staff members, was kidnapped.", GetGlobalFlag(GLinemodeSp));
```

Comment:
@Norgus @DoctorDiablo could any of you guys double-check this for me? I usually handle these pretty easily but this one got me second guessing myself.